bin/inference/pycbc_inference

```diff
@@ -52,6 +52,10 @@ parser.add_argument("--save-backup", action="store_true",
                     default=False,
                     help="Don't delete the backup file after the run has "
                          "completed.")
+parser.add_argument("--cleanup-files", action="store_true",
+                    help="Clean up the temporary backup and checkpoint files")
```

Comment:
It's a little confusing from the wording how this interacts with the `save-backup` option. Maybe change this to `delete-checkpoint`? Or change the help message to just say "checkpoint file"? Or get rid of the `save-backup` option, and just always have it delete the backup file. I'm not sure why you would want the bkup file anyway after the exectuable has finished, since the checkpoint file should be the same.